src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  --hover-link-color: #00acdc;
   margin: 0;
 }
+
+.header {
+  box-sizing: border-box;
```

Comment:
the rule `box-sizing: border-box` must be specified for all elements

Reply:
but in the checklist is written that I am not allowed to use * selector and I should set styles only for elements that require them, what should I do?